Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml

```diff
@@ -6,10 +6,10 @@ starttaskid: "0"
 tasks:
```

Comment:
Please make sure to update this README after all changes are done: https://github.com/demisto/content/blob/master/Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership_README.md

Reply:
[6e27a2f](https://github.com/demisto/content/pull/26334/commits/6e27a2fbc533642e2118f899ab3b10bf3a760ac6)